### PR TITLE
docs: fix markup of arguments

### PR DESCRIPTION
**Pull request by tbm**
This is probably incomplete but a good start.

I don't have a Go environment so this is not tested.

### CODE REVIEW

**Comment by tbm** [comment]
> I don't have a Go environment so this is not tested.

CI will complain if I introduced syntax errors. But I'm worried I deleted some whitesace and words will stick together in the output.  I *think* I've avoided such mistakes though.

File: cmd/mountlib/mount.go [hunk 1]
```diff
@@ -199,7 +199,7 @@ normal drives to be fast and reliable, while cloud storage is far from
 that.  See also [Limitations](#limitations) section below for more
 info
 
-Add "--fuse-flag --VolumePrefix=\server\share" to your "mount"
+Add ` + "`--fuse-flag --VolumePrefix=\\server\\share`" + ` to your ` + "`mount`" + `
```
Comment (ncw) [inline cmd/mountlib/mount.go:202]: I don't think these \ need to be doubled up? I can see them displaying correctly here: https://tip.rclone.org/commands/rclone_mount/ (this is the docs website made from master).

Comment (tbm) [reply cmd/mountlib/mount.go:202]: Without it, Go was complaining about unknown escape sequences.

I don't know Go, but my understanding is that \`...\` is a literal string, so you can put anything in it without escaping.  But "..." is a normal string, so "\s" is seen as the escape sequence for "s", which doesn't exist.

Comment (ivandeex) [reply cmd/mountlib/mount.go]: resolved in the rebased tree

**Comment by ivandeex** [comment]
@tbm
Your branch was rebased on current rclone master.
Please pull the result from github, **do not** force-push now.
Your original work had been saved at https://github.com/ivandeex/rclone/tree/tbm-markup-saved-2021-01-22

Also, in the second patch I propose a more readable solution to the problem.
Please note that `strings.ReplaceAll` was introduced in go 1.12 which is the minimum required version since https://github.com/rclone/rclone/commit/01adee75544dca750d929fdbf9cde0c9b70b5c4d#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721

@ncw
Please review :pray: :cat:

**Review by ncw — APPROVED** [review]
That looks good to me :-)

Please merge @ivandeex

**Pull request merged**